### ISSUE TITLE
Did something go wrong with the 7.3.2 release?

### ISSUE THREAD

**Issue by EdSchouten** [bazelbuild/bazel#24355]
### Description of the bug:

Go to this page:

https://github.com/bazelbuild/bazel/releases/tag/7.3.2

Notice how it links to the following commit:

<img width="122" alt="Screenshot 2024-11-17 at 12 15 43" src="https://github.com/user-attachments/assets/4594c940-3e76-47da-963d-d4e00fb76110">

Click on that commit. Now you end up on this page:

https://github.com/bazelbuild/bazel/commit/56d0e2c64f78a887ce50b2618e57159726a7a208

Notice this spooky banner at the top of the page:

<img width="805" alt="Screenshot 2024-11-17 at 12 16 14" src="https://github.com/user-attachments/assets/939111a9-bf13-4b21-ae97-c9089d603142">

**Comment by fmeum** [comment]
I would guess that the same thing happened with 7.3.1 before the release of 7.3.2 as the release branch also misses the release commit: https://github.com/bazelbuild/bazel/commits/release-7.3.1

**Comment by iancha1992** [comment]
@EdSchouten You're right. It is strange because we have https://github.com/bazelbuild/bazel/commit/7c2350a381a30f50e0fbdd450c699d47561bad67

This is not a big deal. But the release team will look into it. Thanks for letting us know!

cc: @bazelbuild/triage
